src/places/places.controller.ts

```diff
@@ -91,12 +88,19 @@ export class PlacesController {
     }
   }
 
+  @ApiHeader({ name: 'authorization' })
+  @ApiResponse({ type: Place })
   @SetMetadata(MetadataKey.ALLOWED_ROLES, [UserRole.PLACE_MANAGER, UserRole.ADMIN])
   @UseGuards(AuthGuard)
   @Post('/')
-  public async createPlace(@Body() placeDto: CreatePlaceDto): Promise<Place> {
+  public async createPlace(@SessionUserId() userId: string, @Body() placeDto: CreatePlaceDto): Promise<Place> {
     try {
       const place = await this.sequelize.transaction(async (transaction) => {
+        const user = await this.usersService.getUserById(transaction, userId);
+        if (!user || user.role !== UserRole.ADMIN) {
+          throw new CRUDError();
```

Comment:
I think we should add new error for this matter, i.e. `AuthorizationError` and return 403 unauthorized for cases like this one. CRUDError was meant to be an error for general CRUD problems.

Reply:
Definitely we should :) I'll add them and map them to the response